atlas/default_values.py

```diff
@@ -251,7 +251,7 @@ def make_default_config():
     conf["cobining_min_contig_length"] = 2000
     conf["cobining_min_bin_size"] = 200 * 1000
     conf["semibin_options"] = " --recluster --max-node 1 --max-edges 200 "
-    conf['cobinning_separator'] = ":"
```

Comment:
In theory my contigs are named '{sample}_nr' so they are unique. I think I stay with ':' as it's also used by semibin.